src/main/java/com/oltpbenchmark/types/DatabaseType.java

```diff
@@ -45,16 +45,21 @@ public enum DatabaseType {
     ORACLE(true, false),
     SINGLESTORE(true, false),
     SPANNER(false, true),
-    SQLAZURE(true, false),
+    SQLAZURE(true, true, true),
     SQLITE(true, false),
-    SQLSERVER(true, false),
+    SQLSERVER(true, true, true),
     TIMESTEN(true, false),
     PHOENIX(true, true);
 
 
-    DatabaseType(boolean escapeNames, boolean includeColNames) {
+    DatabaseType(boolean escapeNames, boolean includeColNames, boolean loadNeedsUpdateColumnSequence) {
```

Comment:
Per PR feedback, we use this additional DbType feature to consolidate the check for whether we should attempt to update column sequences after loading data.